src/main/java/com/technopolis/server/server/model/User.java

```diff
@@ -0,0 +1,33 @@
+package com.technopolis.server.server.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+@Data
+public class User extends BaseEntity {
+    @Column(name = "username", nullable = false)
+    private String username;
+
+    @Column(name = "first_name", nullable = false)
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+    @Column(name = "email", nullable = false)
```

Comment:
данное поле должно быть уникальным в таблице

```suggestion
    @Column(name = "email", nullable = false, unique = true)
```